Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -5914,7 +5934,30 @@ private void HighlightElement(ElementInfo ElementInfo, bool locateElementByItLoc
                     }
                     if (!string.IsNullOrEmpty(ElementInfo.XPath))
                     {
-                        ElementInfo.ElementObject = Driver.FindElement(By.XPath(ElementInfo.XPath));
+
+                        IList<string> XPaths = ((HTMLElementInfo)ElementInfo).XPathList;
```

Comment:
add condition to execute this block only in case xpathlist is not empty

Reply:
ok